packages/table/src/headers/rowHeader.tsx

```diff
@@ -188,18 +209,45 @@ export class RowHeader extends React.Component<IRowHeaderProps, {}> {
                 </Resizable>
             </DragSelectable>
         );
+
+        return (!isReorderable) ? children : (
+            <DragReorderable
+                key={Classes.rowIndexClass(rowIndex)}
+                locateClick={this.locateClick}
+                locateDrag={this.locateDragForReordering}
+                onReorder={onReorder}
+                onReorderPreview={onReorderPreview}
+                onSelection={onSelection}
+                selectedRegions={selectedRegions}
+                toRegion={this.toRegion}
```

Comment:
(Revisiting: this is probably an argument to refactor `Regions` such that its member functions are declared outside of it.)